packages/cli/src/workflows/workflow.service.ts

```diff
@@ -1,4 +1,4 @@
-import { Container } from 'typedi';
+import { Container, Service } from 'typedi';
```

Comment:
Can we please replace `Container.get` with injected dependencies in this file as well?

Reply:
Planning to do this separately soon! See `Up next` in PR description.